tests/cross_functional/conftest.py

```diff
@@ -141,17 +142,21 @@ def factory(
             kind="secret", namespace=config.ENV_DATA["cluster_namespace"]
         )
         secrets = [
-            "noobaa-root-master-key",
             "noobaa-admin",
             "noobaa-operator",
             "noobaa-db",
             "noobaa-server",
             "noobaa-endpoints",
         ]
-        if is_kms_enabled():
-            secrets = [
-                secret for secret in secrets if secret != "noobaa-root-master-key"
-            ]
+        if (
```

Comment:
what if the cluster is kms enabled and it is running on 4.13? it should have the `noobaa-root-master-key` ?